test/Analysis/PHP/Spec.hs

```diff
@@ -44,5 +46,4 @@ spec = do
 
   where
     fixtures = "test/fixtures/php/analysis/"
-    evaluate = evalPHPProject . map (fixtures <>)
-    evalPHPProject = testEvaluating <=< evaluateProject' ?session (Proxy :: Proxy 'Language.PHP) phpParser
+    evaluate = evaluateProject @'Language.PHP @(PHP.Term Loc) ?session Proxy . map (fixtures <>)
```

Comment:
Pretty happy with how these cleaned up. We’ll talk about the blood sacrifice this required in a moment.